app/controllers/api/public/v1/users_controller.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module Api
+  module Public
+    module V1
+      class UsersController < Api::ApiController
+        def index
+          render json: { users: User.all.as_json(only: %i[id email full_name]) }
+        end
+      end
+    end
+  end
+end
```

Comment:
What is this doing here? Why was it moved to `Public`?
Please revert that change, it will cause conflicts with https://github.com/DFE-Digital/early-careers-framework/pull/341 and more importantly, the users api is not going to be public. We will add authentication to it.

Reply:
Well, it was moved to public for that reason, no authentication)